vms/platformvm/vm.go

```diff
@@ -90,12 +99,13 @@ type VM struct {
 
 	// TODO: Remove after v1.11.x is activated
 	pruned utils.Atomic[bool]
+	cancel context.CancelFunc
```

Comment:
nit: should this be a more specific name, like `gossipCancel`?

Reply:
I think in the future we will probably want to use `cancel` to kill goroutines, so i think this name is fine for now